### PR TITLE
Fix Fuchsia's URLs for Xi and Ledger

### PR DESCRIPTION
**Pull request by kerneis**

### CODE REVIEW

File: docs/docs/crdt-details.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ This document contains a detailed description of the data structures and operati
...
 
-Many of these data structures and operations have been in Xi for a while but they've recently been heavily overhauled and extended as part of [a project](https://github.com/google/xi-editor/issues/250) that added multi-device syncing support via the CRDT merge operation. This was done for [use on the Fuchsia operating system](https://fuchsia.googlesource.com/xi/), where it uses [Ledger](https://fuchsia.googlesource.com/ledger/) to synchronize documents between devices.
+Many of these data structures and operations have been in Xi for a while but they've recently been heavily overhauled and extended as part of [a project](https://github.com/google/xi-editor/issues/250) that added multi-device syncing support via the CRDT merge operation. This was done for [use on the Fuchsia operating system](https://fuchsia.googlesource.com/third_party/xi-editor/), where it uses [Ledger](https://fuchsia.googlesource.com/peridot/+/HEAD/docs/ledger) to synchronize documents between devices.
```
Comment (raphlinus) [inline docs/docs/crdt-details.md]: I think the most appropriate link for Fuchsia xi is `https://fuchsia.googlesource.com/topaz/+/master/bin/xi/` ; the third party one is basically just a mirror of upstream xi and does not contain the front-end.

Comment (kerneis) [reply docs/docs/crdt-details.md]: Done.

**Comment by kerneis** [comment]
PTAL.

**Comment by cmyr** [comment]
Looks good, thanks!

**Pull request merged**